### PR TITLE
Support benchmark OnnxRuntime model

### PR DESCRIPTION
**Pull request by frankfliu**
## Channel for questions ##
Before or while filing an issue please feel free to join our [Slack channel](https://join.slack.com/t/deepjavalibrary/shared_invite/zt-ar91gjkz-qbXhr1l~LFGEIEeGBibT7w) to get in touch with development team, ask questions, find out what's cooking and more!

## Description ##
(Brief description of what this PR is about)

### Essentials ###
Please feel free to remove inapplicable items for your PR.
- [ ] Changes are complete (i.e. I finished coding on this PR)
- [ ] All changes have test coverage:
- [ ] Code is well-documented: 
    - For user-facing API changes, Java doc has been updated. 
    - For new examples, README.md is added to explain the what the example does.
- [ ] To the my best knowledge, [examples](https://github.com/awslabs/djl/tree/master/examples) and [jupyter notebooks](https://github.com/awslabs/djl/tree/master/jupyter) are either not affected by this change, or have been fixed to be compatible with this change

### Changes ###
- [ ] Feature1, tests, (and when applicable, Java doc)
- [ ] Feature2, tests, (and when applicable, Java doc)

## Comments ##
- If this change is a backward incompatible change, why must this change be made?
- Interesting edge cases to note here

### CODE REVIEW

File: examples/src/main/java/ai/djl/examples/inference/benchmark/util/AbstractBenchmark.java [hunk 1]
```diff
@@ -287,7 +288,10 @@ public NDList processInput(TranslatorContext ctx, Object input) {
                         /** {@inheritDoc} */
                         @Override
                         public Object processOutput(TranslatorContext ctx, NDList list) {
-                            return list.get(0).toFloatArray();
```
Comment (zachgk) [inline examples/src/main/java/ai/djl/examples/inference/benchmark/util/AbstractBenchmark.java:290]: Does the call to toFloatArray() not work?

File: onnxruntime/onnxruntime-engine/src/main/java/ai/djl/onnxruntime/engine/OrtNDManager.java [hunk 2]
```diff
@@ -105,13 +105,23 @@ public NDList load(Path path) {
...
 
     /** {@inheritDoc} */
     @Override
     public NDArray ones(Shape shape, DataType dataType) {
-        throw new UnsupportedOperationException("Not supported for ONNX Runtime");
+        int bytes = dataType.getNumOfBytes();
+        int size = Math.toIntExact(bytes * shape.size());
+        ByteBuffer buffer = allocateDirect(size);
+        for (int i = 0; i < size; ++i) {
+            buffer.put((byte) 1);
```
Comment (zachgk) [inline onnxruntime/onnxruntime-engine/src/main/java/ai/djl/onnxruntime/engine/OrtNDManager.java:121]: Is this right? Wouldn't it put the same bytes regardless of the data type?

File: examples/src/main/java/ai/djl/examples/inference/benchmark/util/AbstractBenchmark.java (hunk 1 above)
Comment (frankfliu) [reply examples/src/main/java/ai/djl/examples/inference/benchmark/util/AbstractBenchmark.java:290]: The output dataType may not be float32.

Comment (zachgk) [reply examples/src/main/java/ai/djl/examples/inference/benchmark/util/AbstractBenchmark.java:290]: Then, you could to `.toType(..).toFloatArray()` or `.toArray()`

File: onnxruntime/onnxruntime-engine/src/main/java/ai/djl/onnxruntime/engine/OrtNDManager.java (hunk 2 above)
Comment (frankfliu) [reply onnxruntime/onnxruntime-engine/src/main/java/ai/djl/onnxruntime/engine/OrtNDManager.java]: Fixed.

File: examples/src/main/java/ai/djl/examples/inference/benchmark/util/AbstractBenchmark.java (hunk 1 above)
Comment (frankfliu) [reply examples/src/main/java/ai/djl/examples/inference/benchmark/util/AbstractBenchmark.java:290]: Ort doesn't support toType().

**Review by zachgk — APPROVED** [review]
